core/client/app/mixins/ed-editor-api.js

```diff
@@ -129,7 +129,7 @@ var EditorAPI = Ember.Mixin.create({
 
         $textarea.focus();
         // Tell the editor it has changed, as programmatic replacements won't trigger this automatically
-        this.sendAction('onChange');
+        this.set('value', $textarea.val());
```

Comment:
> I added this.sendAction('onChange') to try to ensure that programmatic changes to the textarea were picked up

There's nothing defined for "onChange" so calling `sendAction('onChange')` here is a no-op.  If you want to send an action outside the `gh-ed-editor` component, then `onChange` needs to be the name of an action.  If you want to do some action inside the editor, `onChange` needs to be in the actions hash, and then the call will be `this.send('onChange');`